api/src/certification/evaluation/domain/services/index.js

```diff
@@ -11,12 +11,14 @@ import * as sharedChallengeRepository from '../../../../shared/infrastructure/re
 import { injectDependencies } from '../../../../shared/infrastructure/utils/dependency-injection.js';
 import * as challengeRepository from '../../../evaluation/infrastructure/repositories/challenge-repository.js';
 import * as certificationAssessmentRepository from '../../../shared/infrastructure/repositories/certification-assessment-repository.js';
+import * as sharedCertificationCandidateRepository from '../../../shared/infrastructure/repositories/certification-candidate-repository.js';
 import * as certificationCourseRepository from '../../../shared/infrastructure/repositories/certification-course-repository.js';
 import * as competenceMarkRepository from '../../../shared/infrastructure/repositories/competence-mark-repository.js';
 import * as complementaryCertificationBadgesRepository from '../../../shared/infrastructure/repositories/complementary-certification-badge-repository.js';
 import * as complementaryCertificationCourseResultRepository from '../../../shared/infrastructure/repositories/complementary-certification-course-result-repository.js';
 import * as flashAlgorithmConfigurationRepository from '../../../shared/infrastructure/repositories/flash-algorithm-configuration-repository.js';
 import * as scoringConfigurationRepository from '../../../shared/infrastructure/repositories/scoring-configuration-repository.js';
+import * as versionRepository from '../../../shared/infrastructure/repositories/version-repository.js';
```

Comment:
```suggestion
import * as sharedVersionRepository from '../../../shared/infrastructure/repositories/version-repository.js';
```

**question:** c'est pas une bonne pratique de scoper un repo "externe" ?

Reply:
Je fix